src/components/test/dropdown-alerts.njk

```diff
@@ -0,0 +1,129 @@
+    <option value="clementine">Clementine</option>
+    <option value="crab apple">Crab apple</option>
+    <option value="currant">Currant</option>
+    <option value="cherry">Cherry</option>
+    <option value="custard apple">Custard apple</option>
+    <option value="coconut">Coconut</option>
+    <option value="cranberry">Cranberry</option>
+    <option value="date">Date</option>
+    <option value="dragonfruit">Dragonfruit</option>
+    <option value="durian">Durian</option>
+    <option value="elderberry">Elderberry</option>
+    <option value="fig">Fig</option>
+    <option value="gooseberry">Gooseberry</option>
+    <option value="grape">Grape</option>
+    <option value="grapefruit">Grapefruit</option>
+    <option value="guava">Guava</option>
+    <option value="honeydew melon">Honeydew melon</option>
+    <option value="jackfruit">Jackfruit</option>
+    <option value="kiwifruit">Kiwifruit</option>
+    <option value="kumquat">Kumquat</option>
+    <option value="lemon">Lemon</option>
+    <option value="lime">Lime</option>
+    <option value="lychee">Lychee</option>
+    <option value="mandarine">Mandarine</option>
+    <option value="mango">Mango</option>
+    <option value="mangosteen">Mangosteen</option>
+    <option value="marionberry">Marionberry</option>
+    <option value="nectarine">Nectarine</option>
+    <option value="orange">Orange</option>
+    <option value="papaya">Papaya</option>
+    <option value="passionfruit">Passionfruit</option>
+    <option value="peach">Peach</option>
+    <option value="pear">Pear</option>
+    <option value="persimmon">Persimmon</option>
+    <option value="plantain">Plantain</option>
+    <option value="plum">Plum</option>
+    <option value="pineapple">Pineapple</option>
+    <option value="pluot">Pluot</option>
+    <option value="pomegranate">Pomegranate</option>
+    <option value="pomelo">Pomelo</option>
+    <option value="quince">Quince</option>
+    <option value="raspberry">Raspberry</option>
+    <option value="rambutan">Rambutan</option>
+    <option value="soursop">Soursop</option>
+    <option value="starfruit">Starfruit</option>
+    <option value="strawberry">Strawberry</option>
+    <option value="tamarind">Tamarind</option>
+    <option value="tangelo">Tangelo</option>
+    <option value="tangerine">Tangerine</option>
+    <option value="ugli fruit">Ugli fruit</option>
+    <option value="watermelon">Watermelon</option>
+    <option value="white currant">White currant</option>
+    <option value="yuzu">Yuzu</option>
+  </select>
+</div>
+
+
+<div class="usa-form-group">
+  <label class="usa-label" id="appointment-time-label" for="appointment-time">Appointment time</label>
+  <div class="usa-hint" id="appointment-time-hint">hh:mm</div>
+  <div class="usa-time-picker usa-input--error">
+    <input class="usa-input" id="appointment-time" name="appointment-time" type="text" aria-describedby="appointment-time-label appointment-time-hint">
```

Comment:
It won't work, but the error class should be on the input.

Reply:
Resolved in 25df410